### PR TITLE
[Term Entry] C++ vector .back() 

### LINKED ISSUE #4984: [Term Entry] C++ vector .back()
### Reason/inspiration (optional)

"We would like a new entry on the `.back()` term entry in Cpp under `Vector` concept. The entry should go in a new file under `content/cpp/concepts/vectors/terms/back/back.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept.
- An `Example` section that provides an example demonstrating the concept in use.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by SaviDahegaonkar**
### Issue Solved
Closes #4984 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/cpp/concepts/vectors/terms/back/back.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+The C++ **`.back()`** method is responsible for accessing the last element in the vector. It allows you to modify the data or element in the vector without actually removing it from the vector. It is mostly used to access the recently element added.
+
+## Syntax
+
+```pseudo
+vectorName.back();
+```
+To use vectors, it is necessary to `#include` the `vector` library as the vector is a part of STL library.
+
+The below example shows the use of `.back()` method in c++ vectors, here `numbers` is a vector which has 5 elements in it and it displays the last element in the vector with the help of the `.back()` method, then the last element in the vector is modifies and the modified value is displayed.
```
Comment (avdhoottt) [inline content/cpp/concepts/vectors/terms/back/back.md:28]: Put this line after the `Example` heading.

File: content/cpp/concepts/vectors/terms/back/back.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+//This is compulsory to include vectors while using vectors.
+
+int main(){
+  std::vector<int>numbers = {10,20,40,50,60};
+  std::cout<<"The last element in the vector is: "<< numbers.back()<< std::endl;
+  numbers.back() = 80;
+  std::cout <<"The last element in the vector after modification is: "<<numbers.back() <<std::endl;
+  return 0;
+  }
+```
```
Comment (avdhoottt) [inline content/cpp/concepts/vectors/terms/back/back.md:44]: Add a line here saying that the following is the output of the above code. You can write it as you want.

File: content/cpp/concepts/vectors/terms/back/back.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+```shell
+The last element in the vector us: 60
+The last element in the vector after modification is: 80
+```
+
+This codebyte example shows that how a last element in the vector in accessed and modified using the `back()` method.
```
Comment (avdhoottt) [inline content/cpp/concepts/vectors/terms/back/back.md:50]: Add this after the Codebyte heading.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @SaviDahegaonkar, I have suggested some changes. Please make them!!

**Comment by SaviDahegaonkar** [comment]
Hello @avdhoottt, Sorry for the late response
I have made the changes :)

Thanks,
Savi

**Pull request closed without merging**